man/systemd-hibernate-resume-generator.xml

```diff
@@ -55,6 +55,13 @@
         supported.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>resume_offset=</varname></term>
+
+        <listitem><para>Takes the byte offset of the swap space from the resume device.
```

Comment:
hmm, this is not correct. the unit is page size. see Documentation/admin-guide/kernel-parameters.txt in the kernel source tree.